gcp/apis/compute/v1alpha2/gkecluster_types.go

```diff
@@ -36,76 +36,127 @@ const (
 	DefaultNumberOfNodes = int64(1)
 )
 
-// GKEClusterParameters specifies the configuration of a GKE cluster.
+// GKEClusterParameters define the desired state of a Google Kubernetes Engine
+// cluster.
 type GKEClusterParameters struct {
-	Addons                     []string          `json:"addons,omitempty"`
```

Comment:
a bunch of these GKE cluster properties have been removed.  What was your criteria for decided what to remove?

Reply:
None of them were actually used. If you look at `clients/gcp` you'll see that they were not referenced when creating a cluster. Exposing fields that we did nothing with seemed like a very misleading experience for our users, so I removed them.